test/test_torch.py

```diff
@@ -17485,7 +17485,6 @@ def test_rdiv(self, device, dtype):
         z = torch.tensor([30 / v.item() for v in x], device=device)
         self.assertEqual(y, z, exact_dtype=False)
 
-    @onlyCPU
     @dtypes(*torch.testing.get_all_dtypes(include_bfloat16=False, include_bool=False, include_complex=False))
```

Comment:
We should test half on CUDA. We should also check if bfloat16 on CUDA was supported or not (it doesn't look like there were good fmod tests before this PR, so as a part of the port we should check the edge cases to make sure we're not regressing anything).

Reply:
the @dtypesIfCPU / @dtypesIfCUDA macros will be helpful here

Reply:
Half has already been included as `include_half=True` is the default argument. `bfloat16` was not supported so no need to change here.